PokemonRampUp/Model/Move.swift

```diff
@@ -8,8 +8,8 @@
 import Foundation
 
 // Move struct represents a Pokemon move
-struct Move: Decodable {
-    let name: String
+struct Move: Decodable, NameContainerProtocol {
+    var name: String
```

Comment:
I had to switch name from `let` to `var` in `Ability`, `TypeElement` and `Move` structs. This needed to be done for `capitalizeFirstLetterOfNameContainers` method in `PokemonJSONToPokemon`. I didn't know who to do it without making it mutable. If there is a better way I am open to hear it.

Reply:
Names are always capitalized. It kind of just bugged me looking at names that are not capitalized. I think it is worth the effort to switch it but, we can see how people feel on demo day